docs/scos/dev/feature-integration-guides/202304.0/install-the-shipment-service-point-feature.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Install the Shipment + Service Points feature
```

Comment:
the included file's name, IG's name, and the title must be identic. In file names, it is "service-point" and the title says "service points"? Chose the proper one and adjust the title or file names accordingly.

Reply:
Fixed file names 👍